groups/groups.yaml

```diff
@@ -57,14 +57,18 @@ groups:
   - email-id: calendar-maintainers@knative.team
     name: calendar-maintainers
     description: |-
-      Folks that can manage Knative calendar
+      Folks that can manage Knative calendar for one-off calendar invites
     settings:
       AllowExternalMembers: "false"
+      WhoCanPostMessage: "ANYONE_CAN_POST"
```

Comment:
Since this group is only intended for ACLs, can we change this to (effectively) NOBODY_CAN_POST? 😁

Reply:
https://knative.slack.com/archives/CU9DGL4FM/p1651791266269549?thread_ts=1651763947.024839&cid=CU9DGL4FM